python/ray/tests/test_output.py

```diff
@@ -528,13 +547,4 @@ def __repr__(self):
 
 
 if __name__ == "__main__":
-    if len(sys.argv) > 1 and sys.argv[1] == "_ray_instance":
```

Comment:
Hmm actually can you just remove this change (it is unrelated to the PR at the end of the day)? While I feel it is 90% okay to change, I wonder if there's a reason why test was written in this way. Maybe cc @wuisawesome knows?

Reply:
feels like it is just historical code

Reply:
I don't know why this was here (my guess is it could've been tied to something with travis ci?). I'm not against removing a special case if it's not needed...

Reply:
I've moved the change to https://github.com/ray-project/ray/pull/23946